integration-tests/features/search/search.feature

```diff
@@ -87,3 +87,19 @@ Feature: Search
         When method get
         Then status 400
         And match $.Message == 'Not a valid language code.'
+
+
+    Scenario: cgov word boundries  #63
+        Given path 'Search', 'cgov', 'en', "hodgkin's disease"
+        And param size = 10
+        When method get
+        Then status 200
+        And match $.results[0].title contains 'Hodgkin'
```

Comment:
How does this test word boundaries? Should it match "non-hodgkin"?  Are there cases where a match should **not** happen?  Are there other terms that should be tested? (e.g. Terms which contain, or start with non alphabetic characters?)

Should there be other tests for other collections and languages? (These are tests for whether the API is doing the right thing, not Elasticsearch.)

Reply:
I will update the scenario names. 

I can expand based on the current two cases to cover ES, and DOC.

It will match non-hodgkin, that is by design. 

the case of 5-fu is a case that terms which contain, or start with non alphabetic characters.